rustsec/src/error.rs

```diff
@@ -1,6 +1,7 @@
 //! Error types used by this crate
 
 use std::{
+    error::Error as ErrorTrait,
```

Comment:
To import this while avoiding a name collision you can do:

```suggestion
    error::Error as _,
```

Reply:
Turns out importing the trait isn't needed at all, given that we're going to use it fully qualified.